lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/data/ashrae_90_1_2019.prototype_inputs.json

```diff
@@ -413,7 +413,7 @@
       "building_type": "Hospital",
       "hvac_system": null,
       "geometry_osm": "ASHRAE9012013Hospital.osm",
-      "hvac_json": "ASHRAE90120102013Hospital.json",
+      "hvac_json": "ASHRAE90120162019Hospital.json",
```

Comment:
@mdahlhausen - This was made as part of #1060, do you know why it was changed back?

Reply:
I'm not sure; we can look at the history of the spreadsheet edits perhaps.  I'd like to move to a different method besides the spreadsheet so it is easier to catch/review these sorts of changes.